spec/models/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { is_expected.to belong_to :user }
+
+  it { is_expected.to have_attached_file(:image) }
+
+  it do
+    is_expected.to validate_attachment_content_type(:image)
+      .allowing('image/png', 'image/gif', 'image/jpg', 'image/jpeg')
+      .rejecting('text/plain', 'text/xml', 'text/html')
+  end
+
+  describe '#created_by?' do
+    let!(:user) { FactoryGirl.create(:user) }
+
+    it 'returns true if post was created by the user' do
+      post = FactoryGirl.create(:post, user: user)
+      expect(post.created_by? user).to be true
```

Comment:
Style/NestedParenthesizedCalls: Add parentheses to nested method call `post.created_by? user`.